frontend/app/config/webpack.config.js

```diff
@@ -439,45 +424,6 @@ module.exports = function (webpackEnv) {
                   isEnvDevelopment &&
                     shouldUseReactRefresh &&
                     require.resolve("react-refresh/babel"),
-                ].filter(Boolean),
-                // This is a feature of `babel-loader` for webpack (not Babel itself).
-                // It enables caching results in ./node_modules/.cache/babel-loader/
-                // directory for faster rebuilds.
-                cacheDirectory: true,
-                // See #6846 for context on why cacheCompression is disabled
-                cacheCompression: false,
-                compact: isEnvProduction,
-              },
-            },
-            // watch and recompile streamlit lib
-            {
-              test: /\.(js|mjs|jsx|ts|tsx)$/,
-              include: /streamlit\/frontend\/lib/,
```

Comment:
We are essentially doing the same here, but with special aliasing. It's not working, so let's remove it.